src/Main.java

```diff
@@ -1,81 +1,53 @@
 
-        // task1
+            //task #1
 
-        byte clientOS = 1;
-        if (clientOS == 0) {
-            System.out.println("Установите версию для iOS по ссылке.");
-        } else {
-            System.out.println("Установите версию для Android по ссылке.");
+        int[] sumCost = {30, 25, 29, 32, 33};
+        int sum = 0;
+        for (int element : sumCost) {
+            sum = sum + element;
         }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей.");
 
-        // task2
+            //task #2
 
-        byte clientOS1 = 0;
-        int clientDeviceYear = 2014;
-        if (clientOS1 == 1 && clientDeviceYear < 2015) {
-            System.out.println("Установите облегченную версию приложения для Android по ссылке.");
-        } else if (clientOS1 == 0 && clientDeviceYear < 2015) {
-            System.out.println("Установите облегченную версию приложения для iOS по ссылке.");
-        }
-        if (clientOS1 == 0 && clientDeviceYear >= 2015) {
-            System.out.println("Установите версию для iOS по ссылке.");
-        } else if (clientOS1 == 1 && clientDeviceYear >= 2015) {
-            System.out.println("Установите версию для Android по ссылке.");
+        int[] cost = {30, 25, 29, 32, 33};
+        int minCost = 40;
+        for (final int current : cost) {
+        if (current < minCost){
+                minCost = current;
+            }
         }
+       System.out.println("Минимальная сумма трат за неделю составила " + minCost + " рублей.");
 
-        //task3
-
-        int year = 1704;
-        if ((year >= 1584) && (year % 4 == 0 && year % 100 != 0 || year % 400 == 0)) {
-            System.out.println("Этот " + year + " год является високосным.");
-        } else {
-            System.out.println("Этот " + year + " год не является високосным.");
+        int maxCost = -1;
+        for (final int current : cost) {
+            if (current > maxCost) {
+                maxCost = current;
+            }
         }
+        System.out.println("Максимальная сумма трат за неделю составила " + maxCost + " рублей.");
 
-        //task 4
+            //task #3
 
-        int deliveryDistance = 47;
-        if (deliveryDistance < 20) {
-            int deliveryDay = 1;
-            System.out.println("Потребуется дней: " + deliveryDay);
-        } else if (deliveryDistance >= 20 && deliveryDistance <= 60){
-            int deliveryDay = 2;
-            System.out.println("Потребуется дней: " + deliveryDay);
-        } else if (deliveryDistance > 60 && deliveryDistance <= 100){
-            int deliveryday = 3;
-            System.out.println("Потребуется дней: " + deliveryday);
-        }else {
-            System.out.println("Доставки, на такое расстояние, не осущевляются.");
-        }
+    int[] midCost = {30,43,44,35,37};
+    int sumMonth = 0;
+    for (int element : midCost){
+        sumMonth = sumMonth + element;
+    }
+        float midSum = sumMonth / 4.0f;
```

Comment:
1) Не понятно почему делишь на 4, если в массиве 5 значений
2) Хардкодить это число вообще не нужно, нужно взять длину массива
```suggestion
        float midSum = sumMonth / midCost.length;
```